langchain/evaluation/run_evaluators/implementations.py

```diff
@@ -144,9 +141,9 @@ def parse(self, parsed_output: Union[str, dict]) -> EvaluationResult:
             parsed_output_ = parsed_output
         return EvaluationResult(
             key=self.evaluation_name,
-            score=parsed_output_.get("score"),
-            value=parsed_output_.get("value"),
-            comment=parsed_output_.get("reasoning"),
+            score=parsed_output_["score"],
+            value=parsed_output_["value"],
```

Comment:
perhaps you should use the get() method because it does not raise a KeyError if the key is not found in the dictionary?